src/Provider/3dTilesProvider.js

```diff
@@ -62,6 +62,28 @@ function $3dTilesIndex(tileset, baseURL) {
 function preprocessDataLayer(layer, view, scheduler) {
     layer.sseThreshold = layer.sseThreshold || 16;
     layer.cleanupDelay = layer.cleanupDelay || 1000;
+    // override the default method, since updated objects are metadata in this case
+    layer.getObjectToUpdateForAttachedLayers = (meta) => {
```

Comment:
you should add a test  will verify the function work as expected
and others clarify / document the data structure used for 3dTiles (as for example, the `object`, the `userData` and the `content` are structured)

Reply:
I reverted part of the userData change.

The point was to store our objects in this dict created by three.js instead of patching the constructed objects (mainly because it's a good pratice and it might improve performance).

See [this comment](https://github.com/iTowns/itowns/pull/719#issuecomment-380376078) for a more info.

Reply:
OK fine,
but what is the structure of the data?

Reply:
>  what is the structure of the data?

I don't understand your question sorry.

Reply:
Can you clarify the structure of objects used for 3dTiles, in itowns

Reply:
I didn't change any the structure or its behavior in this PR. Why are you requesting this?

(also the structure is exactly the one from the 3dtiles spec)